src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -114,10 +116,16 @@ class QuantizationModifier(ScheduledModifier):
         transformer based models such as BERT where the quantized MatMul outputs
         are kept at 32 bits of precision and fake quantizing the outputs harm training
         recovery. Default is True
+    :param activation_bits: Number of bits to use for setting quant min/max values for
+            activations. Default is None, which will quantize activations to 8 bits.
```

Comment:
I think it would be confusing that None means 8 bits (to me it means you don't want to quantize activations at all); we should make it explicit to see from the recipe. Can't we default to number?